CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Enterprise Edition: Implemented the JMX monitoring plugin.
```

Comment:
i am not sure either. it's already in docs, though that it can be somehow distinguished from other features